### ISSUE TITLE
Does anyone know if I can use Geoportal Search Widget with Geoportal Server 2.5?

### ISSUE THREAD

**Issue by kescalera84** [Esri/geoportal-server#273]
Does anyone know if I can use Geoportal Search Widget with Geoportal Server 2.5?

Thanks in advance

**Comment by mhogeweg** [comment]
yes you can! the viewer app in geoportal server 2.5 is essentially a web appbuilder app. you can see it in the   /geoportal/viewer folder in the code base. the AddToMap widget there connects the search results to the map viewer. 

Drop the GeoportalSearch widget into the ```viewer/widgets``` folder and update config.json in the viewer to include a reference to this widget:
```
      {
        "uri": "widgets/GeoportalSearch/Widget",
        "version": "2.0.1",
        "position":{
          "left": 55,
          "top": 55
        }
      },
```

**Comment by kescalera84** [comment]
Sorry, I didn't give enough details, I have Geoportal Server Catalog 2.5 Installed and working, I saw that it comes with a map viewer created with Web AppBuilder wich includes Geoportal Search widget which comes configured with two 1.x Geoportal Servers as an example, the widget works well with those Geoportals, but I cannot get to configure it with my own Geoportal Server 2.5, maybe it is me, but I don't know wich url I should suply in the widget config file:

{
  "catalogs": [
      {
        "name": "Geoportal Sandbox",
        "url": "http://gptogc.esri.com/geoportal/rest/find/document"
      },
      {
        "name": "California Geoportal",
        "url": "http://portal.gis.ca.gov/geoportal/rest/find/document"
      },
      {
        "name": "USGIN Geoportal",
        "url": "http://catalog.usgin.org/geoportal/rest/find/document"
      },
      {
        "name": "KEVO'S Geoportal",
        "url": "http://arcgiswin.esri.pa:8080/geoportal/opensearch?f=georss"  ????????????
	  }  
  ],
      
  "recordsPerPage": 10,
  "showAllAddedLayers": true
}



I have been testing most of the options described here: https://github.com/Esri/geoportal-server-catalog/blob/master/geoportal/doc/api.txt

Any help would be appreciated

**Comment by ulenkar** [comment]
kescalera84 - did you find any solution? I'm facing the same problem..

**Comment by mhogeweg** [comment]
hi @ulenkar, did you see my earlier comment? You can drop the search widget in the viewer app and it will work just like it does in a standalone web appbuilder app.

@kescalera84, to register a Geoportal Server 2.5.x in the widget, follow the below approach:

```
      {
        "name": "Geoportal 2.5.0",
        "url": "http://geoss.esri.com/geoportal2/rest/metadata/search?gptVersion=2",
        "gptVersion": 2
      }
```

**Comment by ulenkar** [comment]
I tried as you mentioned before but it didn't worked. But then I downloaded a new version of the widget 
and it finally works :)
